sdk/core/azure-core-tracing-opentelemetry/setup.py

```diff
@@ -58,7 +58,7 @@
     ],
     python_requires=">=3.5.0",
     install_requires=[
-        'opentelemetry-api==0.3a0',
+        'opentelemetry-api==0.4a0',
         'azure-core<2.0.0,>=1.0.0',
```

Comment:
Why ==?

Reply:
We know already that 0.5.0 will be a broken one....